packages/orbit-components/src/primitives/IllustrationPrimitive/__tests__/index.test.js

```diff
@@ -24,11 +24,21 @@ describe("IllustrationPrimitive", () => {
 
     const img = screen.getByRole("img");
 
-expect(img).toHaveStyle({ maxHeight: "90px" });
+    expect(img).toHaveStyle({ maxHeight: "90px" });
     expect(screen.getByAltText("Alternative text")).toBeInTheDocument();
     expect(screen.getByTestId("test")).toBeInTheDocument();
     expect(img).toHaveAttribute("src", URL);
     expect(img).toHaveAttribute("srcset", expect.stringContaining(URL_RETINA));
     expect(img).toHaveStyle({ marginBottom: defaultTheme.orbit.spaceSmall });
   });
+
+  it("should have empty alt", () => {
+    render(<IllustrationPrimitive name="Accommodation" alt="" />);
+    expect(screen.getByAltText("")).toBeInTheDocument();
```

Comment:
Reminder: you don't have to do the `expect(...).toBeInTheDocument()` when you're using `GetBy` because `GetBy` queries themselves already throw when they don't find the element. You only need to use it when you're using `QueryBy`.

We have a ton of these in our codebase and it's nothing something you need to change here, we can later clean them up with a codemod, I'm just saying this so that you know for future.